GoogleForCreators/web-stories-wp
# New Story Details Modal: Open Checklist

## Context

Checklist functionality for new story details modal feature. 

## Summary

The new story details modal has a button to go to the checklist. This wires that up. 


## Relevant Technical Choices

I'm reusing the functionality from the `reviewChecklistModal` - made a new ticket (#10470) to clean this up after feature flag is removed to avoid having to rewire existing functionality. 

## To-do

(See https://github.com/GoogleForCreators/web-stories-wp/issues/10115 for full list)

## User-facing changes

N/A - feature flag

## Testing Instructions

- Set feature flag `enableUpdatedPublishStoryModal` to true and click 'publish' on a story, should open the new story details modal. Now click on "checklist", story details modal should close and checklist should open. 
- If the checklist was open when you clicked publish and opened the story details modal, the story details modal should close and checklist should persist open.




This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially addresses #10115
